### PR TITLE
Allow FormFieldCheckDependentController to check all dependent boxes

### PR DESCRIPTION
**Pull request by rnestler**
Configured with a stimulus value:
https://stimulus.hotwired.dev/reference/values

### CODE REVIEW

File: app/javascript/controllers/form_field_check_dependent_controller.js
Comment (rnestler) [inline app/javascript/controllers/form_field_check_dependent_controller.js:1]: I couldn't find any existing spec for this in core. I guess it isn't used at all in core?

Comment (njaeggi) [reply app/javascript/controllers/form_field_check_dependent_controller.js:1]: There is no use in core for this, the only use is in the SAC wagon, specs right here: https://github.com/hitobito/hitobito_sac_cas/blob/master/spec/features/memberships/sac_terminate_membership_wizard_spec.rb#L70-L109

Comment (rnestler) [reply app/javascript/controllers/form_field_check_dependent_controller.js:1]: Then we can merge it like this.

**Review by njaeggi — APPROVED** [review]
LGTM

**Pull request merged**